### PR TITLE
Added project token and removed need for price feed

### PR DESCRIPTION
**Pull request by nicszerman**
Our tokens are now trading on WAGMI liquidity pools, so we removed the price computation and disabled the parameter "misrepresentedTokens".

We are including the fields below that have been updated.

The wagmi liquidity pools:
ARKKs at 0x56742D48Fd18C236aCEd6317c192eCFB2d4bd792 (https://app.wagmi.com/dashboard/pools/404?chain=2222)
SPYs at 0xeA55Df929438f8A1af003A59381ba57aB2f3fb1f https://app.wagmi.com/dashboard/pools/406?chain=2222

##### Current TVL:
About $400-$500 USD

##### Short Description (to be shown on DefiLlama):
The stock market, onchain.

##### Token address and ticker if any:
ticker, address:
SPYs, https://kavascan.com/token/0x75B5DACEc8DACcb260eA47549aE882513A21CE01/token-transfers
ARKKs, https://kavascan.com/token/0x2E4763AdBEe00D5eB3089ec25973599c0e62dD07/token-transfers

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Synthetics

##### methodology (what is being counted as tvl, how is tvl being calculated):
Value in USD of onchain circulating stocks.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/sailingprotocol exports TVL: 
        
 
 
    kava                      0
    
    total                    0

**Comment by nicszerman** [comment]
The adapter is not correctly calculating the tokens TVL, probably because they are not indexed. How can we add our tokens to the price index?

**Comment by g1nt0ki** [comment]
@nicszerman checked the pools, they have a tvl under $10? 

For us, to price tokens from uni v3 pool, that is very little liquidity, need at least 5000$ swaps with less than 5% move

**Comment by llamatester** [comment]
The adapter at projects/sailingprotocol exports TVL: 
        
 
 
    kava                      449.68546310393293
    
    total                    449.68546310393293

**Comment by nicszerman** [comment]
I'm reading prices from the liquidity pools and adding those unindexed tokens in terms of usdt..

**Comment by llamatester** [comment]
The adapter at projects/sailingprotocol exports TVL: 
        
 
 
    kava                      445.8235641941919
    
    total                    445.8235641941919

**Comment by llamatester** [comment]
The adapter at projects/sailingprotocol exports TVL: 
        
 
 
    kava                      445.8235641941919
    
    total                    445.8235641941919

**Comment by nicszerman** [comment]
Done :) the price of each token is fetched from the unindexed liquidity pools. It would be nice to be able to write token adapters, that will be increasingly useful as web3 is adopted.

**Comment by llamatester** [comment]
The adapter at projects/sailingprotocol exports TVL: 
        
 
 
    kava                      445.8235641941919
    
    total                    445.8235641941919

**Comment by g1nt0ki** [comment]
hi @nicszerman sorry, but this is not much better, this is rwa assets right? is there no on chain oracle where we can read price from? whether we compute the price on our coins server, or you do it each time in tvl adapter, the pool still has very low liquidity right?

**Comment by nicszerman** [comment]
I do not know of an onchain oracle for ARKK, and tomorrow we’re adding
dozens more tickers, for which most also don’t have onchain oracles.
1. These pools have little liquidity but any user can arbitrage them if the
price is not accurate.
2. Alternatively we can fetch price offchain for the tickers without enough
liquidity.

Thank you. Let me know your thoughts.

**Comment by nicszerman** [comment]
One possible enhancement is averaging the pool price over 24 hours.

**Comment by llamatester** [comment]
The adapter at projects/sailingprotocol exports TVL: 
        
 
 
    kava                      445.3781860081838
    
    total                    445.3781860081838

**Comment by llamatester** [comment]
The adapter at projects/sailingprotocol exports TVL: 
        
 
 
    kava                      445.3781860081838
    
    total                    445.3781860081838

**Comment by nicszerman** [comment]
Hey @g1nt0ki , I moved forward with the idea above of averaging over 24 hours to address the low pool liquidity. Thank you very much for your reviewing the PR :)

**Comment by nicszerman** [comment]
Price estimation could be further improved by using geometric mean instead of arithmetic mean.

**Comment by g1nt0ki** [comment]
@nicszerman even if you take the average, it is still low liquidity, compared to this, I prefer pulling the price off chain, does your protocol maintain a rest endpoint that returns the prices?

**Comment by nicszerman** [comment]
Yes we do. Shall I use it? We would be unable to use the timetravel to
build the graph, right?

**Comment by llamatester** [comment]
The adapter at projects/sailingprotocol exports TVL: 
        
 
 
    kava                      445.3781860081838
    
    total                    445.3781860081838

**Comment by llamatester** [comment]
The adapter at projects/sailingprotocol exports TVL: 
        
 
 
    kava                      449.2362268770559
    
    total                    449.2362268770559

**Comment by llamatester** [comment]
The adapter at projects/sailingprotocol exports TVL: 
        
 
 
    kava                      449.68546310393293
    
    total                    449.68546310393293

**Comment by nicszerman** [comment]
Done.

File: projects/sailingprotocol/index.js [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+    "address": "0x2E4763AdBEe00D5eB3089ec25973599c0e62dD07",
+    "ticker": "ARKK",
+    "sufficientLiquidityForDefiLlamaIndexer": false,
+  }
+];
+
+async function tvl(_0, blockNumber, _2, { api }) {
+  for (const token of tokens) {
+    const tokenTotalSupply = await api.call({ target: token.address, abi: 'erc20:totalSupply' });
+    api.add(token.address, tokenTotalSupply);
```
Comment (g1nt0ki) [inline projects/sailingprotocol/index.js:20]: one minor fix, this line can be disabled as you are adding usd value in the next line

Comment (g1nt0ki) [reply projects/sailingprotocol/index.js]: updated it

**Comment by llamatester** [comment]
The adapter at projects/sailingprotocol exports TVL: 
        
 
 
    kava                      448.154612309588
    
    total                    448.154612309588

**Pull request merged**